model-optimizer/setup.py

```diff
@@ -92,6 +91,11 @@ def find_package_modules(self, package, package_dir):
               'mo = mo.__main__:main',
            ],
       },
+      package_data={
```

Comment:
One more file is missing: `mo/front/caffe/proto/mo_caffe.proto`.

Please, add a test to compare content of wheel package and files built with CMake.

Reply:
May be more files are missing, but this one which I was able to find quickly.

Reply:
`mo_caffe.proto` is not used by MO because there are generated `caffe_pb2.py` files, isn't it?

Reply:
It is not used, but we still deliver this file as part of regular installer to allow users re-generate the `caffe_pb2.py` themselves. So I would request to add this file to a pip distribution as well.